api/HTMLMediaElement.json

```
@@ -1630,10 +1630,10 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/media.html#event-media-loadstart",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "32"
```

I'm sure the "loadstart" was supported earlier than this just from memory.

---

https://trac.webkit.org/changeset/36893/webkit is the patch that introduced the "loadstart" event. That was before the release of Chrome 1 by a few months, so let's assume:

```suggestion
              "version_added": "1"
```